addons/website_sale_collect/views/templates.xml

```diff
@@ -39,4 +39,58 @@
         </xpath>
     </template>
 
+    <template id="delivery_location_selector" inherit_id="website_sale.delivery_address_row">
+        <div id="delivery_address_row" position="inside">
+            <div name="o_pickup_location" class="d-none one_kanban col-md">
+                <div name="find_store_btn" class="row row-cols-3 g-1">
```

Comment:
Try to replicate the layout of `.o_portal_address_row`. Here you are not using `cols` inside your `.row`, it will inevitably lead to layout issues. IMO you can remove `.row-cols-3` and wrap `.o_wsale_find_store` inside `<div class="col-md-4">`. To test 👍